app/vue/addons.js

```diff
@@ -1,8 +0,0 @@
-// import deprecate from 'util-deprecate';
-// import '@storybook/addon-actions/register';
-// import '@storybook/addon-links/register';
```

Comment:
Please remove from `package.json`s all the dependencies that are no more used

Reply:
Removed `json5` and `find-cache-dir`. Didn't find others

Reply:
`@storybook/addon-actions` & `@storybook/addon-links`

Reply:
Probably `util-deprecate` as well (if there are usages, they can be removed)

Reply:
> `@storybook/addon-actions` & `@storybook/addon-links`

Were only in app/react -> done

> util-deprecate

done